src/components/fields/field-input-tags/field-input-tags.component.tsx

```diff
@@ -28,21 +30,33 @@ const InputTagsFieldPure: FC<Props & RouterProps> = ({
   label,
   form: { setFieldValue },
   language,
-  isOnlyOneSelectedLanguage
+  isOnlyOneSelectedLanguage,
+  match: { params }
 }) => {
   const tagNodes = _.get(field, 'value', []).map(item => item);
   return (
     <div className="px-2">
-      <label className="fdk-form-label w-100 position-relative" htmlFor={field.name}>
-        {showLabel ? label : null}
-        {!!language && !isOnlyOneSelectedLanguage && <span className="language-indicator">{language}</span>}
-        <TagsInput
-          value={tagNodes}
-          className="fdk-reg-input-tags"
-          inputProps={{ placeholder: '' }}
-          onChange={tags => setFieldValue(_.get(field, 'name'), tags)}
-        />
-      </label>
+      <Can I="edit field" of={{ __type: 'Field', publisher: params.catalogId }}>
```

Comment:
kanskje vi kan holde oss med rettigheter til Publisher type? da slipper to du å lage flere typer som Field og Concept
```
<Can I="add concept" to={{ __type: 'Publisher', publisher: params.catalogId }}>
<Can I="edit field" in={{ __type: 'Publisher', publisher: params.catalogId }}>
```

Reply:
Ikke helt enig. '__type' her er typen i forhold til det som er definert i 'abilities'. Vi kan egentlig ha så mange typer vi vil, dette er abstrahert bort fra brukerrollen. Det gjør at vi kan konfigurere fritt hvilke 'abilities' en bruker har, på et sted. I tillegg blir det mest mulig lesbart i koden.

Reply:
Akkurat. Vi kan bruke hvilke abstrakssjonsnivå vi vil. Eller det som passer best til kravene vår. 
På ett tidspunkt må vi bestemme, hva passer best:
```
        can('create', 'Concept', { publisher: resourceRole.resourceId });
        can('view', 'StatusBar', { publisher: resourceRole.resourceId });	        
        can('edit', ['Language', 'Field'], { publisher: resourceRole.resourceId });
```
eller 
```
 can('edit concept', 'Publisher', { publisher: resourceRole.resourceId });
```

Den er greit, du kan ombestemme seg når du vil :)